docs/reference/faqs.md

```diff
@@ -75,6 +75,16 @@ The helm chart supports instantiating both consoles and engines.  Ingress to the
 Yes, just like Docker, you will be able to use Podman for container orchestration.
 </details>
 
+<details>
+  <summary>Why does Ping recommand K8s vs docker?</summary>
+
+<br>&emsp;1. Docker or a pure container solution like ECS by itself is generally not as robust or resilient as a K8s environment.  That being said, managed Docker services like ECS can provide some of the kubernetes trappings, but that locks you into AWS only and you would have a slightly different experience at Google, Azure, etc.  Kubernetes, even managed like EKS, provide more flexibility and portability</br>
+<br>&emsp;2. It is the model we use for our SaaS offerings, so support, sales, etc. are more familiar with that model</br>
+<br>&emsp;3. Orchestration among multiple applications and services is native to Kubernetes, a bit of an add-on with Container-only services</br>
+<br>&emsp;4. Workload management using K8s primitives (HPA, etc.)</br>
+<br>&emsp;5. Management through IaaC principles using Helm Charts and values files</br>
```

Comment:
Management through Infrastructure-as-Code principles using Helm Charts and Values files.